### PR TITLE
Fix floating point comparison

### PR DESCRIPTION
**Pull request by avalentino**
- [ ] Tests added 
 - [X] Tests passed 
 - [X] Passes ``git diff origin/master **/*py | flake8 --diff`` 
 - [ ] Fully documented

### CODE REVIEW

File: trollsched/tests/test_spherical.py [hunk 1]
```diff
@@ -185,7 +185,7 @@ def test_intersection(self):
         lon, lat = arc1.intersection(arc2)
 
         self.assertTrue(np.allclose(np.rad2deg(lon), 5))
-        self.assertEquals(np.rad2deg(lat), 5.0575148968282093)
+        self.assertTrue(np.allclose(np.rad2deg(lat), 5.0575148968282093))
```
Comment (djhoese) [inline trollsched/tests/test_spherical.py:188]:
```suggestion
        np.testing.assert_allclose(np.rad2deg(lat), 5.0575148968282093)
```

**Comment by djhoese** [comment]
I updated your change to use `np.testing.assert_allclose` but it looks like some unrelated tests are also failing. I'm not sure the last time these tests were run with modern dependencies. They are pretty significant failures.

@pnuu @mraspaud any idea what's going on?

**Comment by pnuu** [comment]
The tests passed less than a month ago for https://github.com/pytroll/pytroll-schedule/pull/61 . No idea (yet) what's has changed since and why the change in this PR is now necessary. I'll have a closer look on monday if I remember and have the time.

**Comment by pnuu** [comment]
Oh, right. the failing tests are fixed in #61 which hasn't been merged as no-one has had time to review it.

**Comment by mraspaud** [comment]
I just merged #61, re-running the tests now

**Pull request merged**